python/torch_mlir/extras/fx_importer.py

```diff
@@ -2401,3 +2360,98 @@ def _ref_finalizer(self, ref_id: int):
     "torch.aten.sub.Tensor": "torch.aten.sub.Scalar",
     "torch.aten.floor_divide": "torch.aten.floor_divide.Scalar",
 }
+
+
+NODE_CANONICALIZE: Dict[TorchOpOverload, Callable] = {}
+
+
+def register_canonicalize(op: TorchOpOverload):
+    def wrapper(func):
+        global NODE_CANONICALIZE
```

Comment:
You don't need this of just reading. Having it here is a strong signal to the reader that you intend to update the global, so better to not leave it out.

Reply:
done